tests/test_text_pair_classification.py

```diff
@@ -52,7 +57,7 @@ def test_text_pair_classification_pipeline(
     pipeline, path = text_pair_classification_pipeline
     result = pipeline.run()
     path.cleanup()
-    np.testing.assert_almost_equal(result["accuracy"]["accuracy"], 0.0789473)
-    np.testing.assert_almost_equal(result["f1__average_macro"]["f1"], 0.0133037)
-    np.testing.assert_almost_equal(result["precision__average_macro"]["precision"], 0.0071770)
-    np.testing.assert_almost_equal(result["recall__average_macro"]["recall"], 0.0909090)
+    np.testing.assert_almost_equal(result["accuracy"]["accuracy"], 0.1538461)
```

Comment:
maybe we should also add places kwargs to the assert_almost_equal with the value of like 3? (by default it is 7)

Reply:
Agree, but it's not in the scope of this PR. I will transfer it to a separate issue.

Reply:
Added to #117